backend-shared/apis/managed-gitops/v1alpha1/gitopsdeploymentmanagedenvironment_types.go

```diff
@@ -29,6 +29,7 @@ type GitOpsDeploymentManagedEnvironmentSpec struct {
 	APIURL                     string `json:"apiURL"`
 	ClusterCredentialsSecret   string `json:"credentialsSecret"`
 	AllowInsecureSkipTLSVerify bool   `json:"allowInsecureSkipTLSVerify"`
+	CreateNewServiceAccount    bool   `json:"createNewServiceAccount,omitempty,default=false"`
```

Comment:
I think the `default=false` can be removed: if `createNewServiceAccount` is not specified, it will already be false as bools default to false.

Reply:
I think I mistakenly added that.  Done